front/app/containers/SearchPage/SearchPage.tsx

```diff
@@ -504,11 +502,29 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
   };
 
   componentDidMount() {
+    let searchTerm = new URLSearchParams(this.props.location.search).getAll("q")
+    if (this.props.location.search) {
```

Comment:
I think this should test UrlSearchParams(...).has('q') instead of if (this.props.location.search)

Reply:
Agreed, and changed!